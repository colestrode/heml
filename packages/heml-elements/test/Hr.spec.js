const render = require('@heml/render')
const jest = require('jest')

jest.mock('../build/Style', () => {
  return {
    render: () => '<mock-style></mock-style>'
  }
})

const HR = require('../build/Hr').default;

test('renders with attrs', () => {
  const attrs = {
    margin: '1px 2px 3px 4px',
    fluff: 'floof'
  }

  expect(render.renderElement(HR, attrs)).toMatchSnapshot()
})
