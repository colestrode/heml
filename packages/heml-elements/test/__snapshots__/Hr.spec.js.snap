// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with some attrs 1`] = `
"<div margin=\\"1px 2px 3px 4px\\" fluff=\\"floof\\" class=\\"hr\\">
  START_CONDITIONmso | IEEND_CONDITION
    <table class=\\"hr__table__ie\\" role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\"><tr><td>
  END_COMMENT_CONDITIONAL
  <table class=\\"hr__table\\" role=\\"presentation\\" border=\\"0\\" align=\\"center\\" cellpadding=\\"0\\" cellspacing=\\"0\\" width=\\"100%\\" style=\\"table-layout: fixed;\\"><tr class=\\"hr__row\\"><td class=\\"hr__cell\\" width=\\"100%\\" align=\\"left\\" valign=\\"top\\">&nbsp;</td></tr></table>
  START_CONDITIONmso | IEEND_CONDITION
    </td></tr></table>
  END_COMMENT_CONDITIONAL
  </div>"
`;
